internal/workflow/processing.go

```diff
@@ -693,5 +693,10 @@ func (w *ProcessingWorkflow) transferAM(sessCtx temporalsdk_workflow.Context, ti
 		return err
 	}
 
+	// Delete Transfer.
+	_ = temporalsdk_workflow.ExecuteActivity(sessCtx, am.DeleteTransferActivityName, am.DeleteTransferActivityParams{
```

Comment:
I think the `sessCtx` activity options are too generous here, there's no timeouts and I'm not sure about the retry policy.  I would use `actOpts := withActivityOptsForRequest(sessCtx)` or something similar.